google-cloud-apis/src/com/google/cloud/tools/intellij/cloudapis/AddCloudLibrariesDialog.java

```diff
@@ -113,7 +112,7 @@ protected void doOKAction() {
       CloudApiManagementConfirmationDialog managementDialog =
           new CloudApiManagementConfirmationDialog(
               getSelectedModule(), cloudProject, apisToEnable, apisNotEnabled, roles);
-      DialogManager.show(managementDialog);
+      managementDialog.show();
```

Comment:
just curious - why was this one needed?

Reply:
Not the best idea to use DialogManager from a very specific package - it's from `git4idea`, there are not `DialogManager` in the core platform, you can just show the dialog itself. Cleaned up as I noticed this wierd import.

Reply:
I noticed that it came from git4idea. I thought something was up with my Gradle sync. sounds good